service-workers/service-worker/resources/update-during-installation-worker.py

```diff
@@ -1,11 +1,13 @@
 import time
 
+from six import PY3
+
 def main(request, response):
-    headers = [('Content-Type', 'application/javascript'),
-               ('Cache-Control', 'max-age=0')]
+    headers = [(b'Content-Type', b'application/javascript'),
+               (b'Cache-Control', b'max-age=0')]
     # Add timestamp to the worker so update() finds a new worker every time.
-    body = '''
+    body = u'''
 // %s
 importScripts('update-during-installation-worker.js');
-    '''.strip() % time.clock()
+    '''.strip() % (time.perf_counter() if PY3 else time.clock())
```

Comment:
I'm concerned about this. The original implementation here might be problematic already: generally speaking, only the deltas between two `time.clock()` or `time.perf_counter()` make sense to measure elapsed wall clock time. The absolute values of these functions are not very useful as the reference points are not well defined.

For the purpose of making the script different each time, let's simply plug in `random.random()` here. And don't forget to update the comment above.